caravel/assets/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "caravel",
-  "version": "0.1.0",
+  "version": "0.10.1",
```

Comment:
@vera-liu  - you may create the test-package.json and then reference it in the test_config.py and maybe create a similar way to do it in js.